backend/server.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const app = express();
+app.listen(3001, () =>{
+  console.log('listing to port 3001');
+});
+
+app.get('/',(req,res)=>{
+  res.send('root file');
+});
+
+app.post('/api/auth/tokens',(req,res) =>{
+  res.send({token : 'J12KL3H1F'})
+});
+app.post('/api/auth/revoke',(req,res) =>{
+  res.send({response : 'revoked!'});
+});
+
+app.get('/api/account/:id',(req,res) =>{
```

Comment:
api/account, we know the user based on the token

Reply:
fixed!